File: PermMissingElem.py
# you can write to stdout for debugging purposes, e.g.
# print "this is a debug message"

def solution(A):
    # write your code in Python 2.7
    sum = 0
    for i in A:
        sum = sum + i

    leng = len(A)
    total = (1+leng+1) * (leng+1) / 2;

    return total - sum

File: Time Complexity/PermMissingElem.cpp
// you can use includes, for example:
// #include <algorithm>

// you can write to stdout for debugging purposes, e.g.
// cout << "this is a debug message" << endl;

int solution(vector<int> &A) {
    // write your code in C++11 (g++ 4.8.2)
    int sum = 0;
    for (auto i : A)
    {    
        sum=sum+i;
    }
    auto len = A.size();
    int total = (1+len+1)*(len+1) / 2;
    return total - sum;
}